src/styles/mixins.scss

```diff
@@ -0,0 +1,13 @@
+@mixin rotate-animation($duration, $timing-function) {
+  animation: hand-move $duration $timing-function infinite;
```

Comment:
The animation name should match the mixin name or be passed as a parameter to the mixin to maintain consistency and avoid confusion. Consider using the mixin name as the animation name or adding an `$animation-name` parameter to the mixin.